crates/matrix-sdk/src/room/mod.rs

```diff
@@ -367,6 +371,32 @@ impl Room {
         (drop_guard, receiver)
     }
 
+    /// Subscribe to updates about users who are in "pin violation" i.e. their
+    /// identity has changed and the user has not yet acknowledged this.
+    ///
+    /// The returned receiver will receive a new vector of
+    /// [`IdentityStatusChange`] each time a /keys/query response shows a
```

Comment:
For completeness, if your own identity trust level changes it could trigger trust changes of other user identities (even if no keys/query has happened).

Reply:
Fixed in 4cd070809c